scan_filter_chain:
#- name: range
#  type: LaserScanRangeFilter
#  params:
#    use_message_range_limits: false
#    lower_threshold: 0.1
#    upper_threshold: 3.0
#    lower_replacement_value: -.inf
#    upper_replacement_value: .inf

#- name: shadows
#  type: ScanShadowsFilter
#  params:
#    min_angle: 15
#    max_angle: 165
#    neighbors: 2
#    window: 2

- name: mrvk_laser_filters
  type: MrvkLaserFilters
  params:
    point_speed_filter:
      enabled: True
      point_speed_max_: 3.0 # [m/s]
    isolated_points_filter:
      enabled: True
      min_accept_size: 3
    transient_points_filter:
      enabled: True
      num_prev_frames: 2